src/app/utils/custom-errors.ts

```diff
@@ -0,0 +1,23 @@
+export class ConflictError extends Error {
```

Comment:
Might want to refactor them into their own module's error. For example, `ConflictError` can probably live inside `modules/submission/submission.errors.ts`, whilst `WebhookValidationError` can live inside `modules/webhooks/webhook.errors.ts`.

The PR for `ApplicationError` has also been merged in, can see #195

Reply:
done.